### DISCUSSION TITLE
[Release notes] Always show package name

### DISCUSSION THREAD

**Discussion by MartijnLeijssen-TomTom** [renovatebot/renovate#22830] [Suggest an Idea]
### Tell us more.

Currently the `Release Notes` section in an PR (if available) shows only the repo name when there is only 1 package in the repository. See the image below
![image](https://github.com/renovatebot/renovate/assets/46988880/c1ec0453-f1b8-4c7a-a42e-e1f5668b1073)

Most of the time this works pretty well, but in my use case we often have package names that doesn't match with the repository name, it would be useful to have the name that is mentioned in the table can also be found in the release notes. 
![image](https://github.com/renovatebot/renovate/assets/46988880/8406b823-b9be-4e45-a427-a791f39c0a7e).

It looks like this was [explicitly implemented](https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/pr/body/changelogs.ts#L27) as part of [showing release notes from a monorepo](https://github.com/renovatebot/renovate/pull/12543). I would like unconditionally add the  ` (${upgrade.depName})`, would this be acceptable? I don't mind making this change myself!

**Comment by rarkins** [comment]
Sure, please go ahead

**Reply by HonkingGoose** [reply]
- #22868
